plugins/sigma/tests/tests.cpp

```diff
@@ -458,9 +461,9 @@ TEST(real example) {
   auto selection1 = R"__(CommandLine ni "7z.exe a -v500m -mx9 -r0 -p")__"s;
   auto selection2a = R"__(ParentCommandLine ni "wscript.exe" && ParentCommandLine ni ".vbs")__"s;
   auto selection2b = R"__(CommandLine ni "rundll32.exe" && CommandLine ni "C:\Windows" && CommandLine ni ".dll,Tk_")__"s;
-  auto selection3 = R"__(ParentImage ni "\rundll32.exe" && ParentCommandLine ni "C:\Windows" && CommandLine ni "cmd.exe /C ")__"s;
+  auto selection3 = R"__(ParentImage == /\rundll32.exe$/ && ParentCommandLine ni "C:\Windows" && CommandLine ni "cmd.exe /C ")__"s;
   auto selection4 = R"__(CommandLine ni "rundll32 c:\windows\\" && CommandLine ni ".dll ")__"s;
-  auto specific1 = R"__(ParentImage ni "\rundll32.exe" && Image ni "\dllhost.exe")__"s;
+  auto specific1 = R"__(ParentImage == /\rundll32.exe$/ && Image == /\dllhost.exe$/)__"s;
```

Comment:
Out of curiosity: is the `\` now triggering an escape sequence, i.e., is the first character of `/\rundll32.exe$/` a `\r` or `r`? I'd suggest to add a unit test that confirms the expected behavior.